modeling-cmds/src/def_enum.rs

```diff
@@ -136,6 +136,27 @@ define_modeling_cmd_enum! {
             pub tolerance: LengthUnit,
         }
 
+        /// Command for lofting sections to create a solid
+        #[derive(
+            Debug, Clone, Serialize, Deserialize, JsonSchema, ModelingCmdVariant,
+        )]
+        pub struct Loft {
+            /// The closed section curves to create a lofted solid from.
+            /// Currently, these must be Solid2D's
+            pub section_ids: Vec<Uuid>,
+            /// Degree of the interpolation.  Pass 2 for quadratic, and 3 for cubic interpolation in the V direction.
```

Comment:
Are these the only valid values? If so this should be an enum, like this [playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=62deb3ab15f40aa2b9c348c21f5c936b)

Reply:
No, any value is valid except I suppose 0.  So this should be kept as the integer I have